Source/carthage/Extensions.swift

```diff
@@ -57,19 +58,28 @@ extension GitURL: ArgumentType {
 
 /// Logs project events put into the sink.
 internal struct ProjectEventSink: SinkType {
+	let colorOptions: ColorOptions
```

Comment:
Can you make this `private`? I don't think consumers of this sink type should need to read this property.